cli/index.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var CryptoJS = require("crypto-js");
+var FileSystem = require("fs");
+const Yargs = require('yargs');
+
+const SCRIPT_URL = 'https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.9-1/crypto-js.min.js';
+const SCRIPT_TAG = '<script src="' + SCRIPT_URL + '" integrity="sha384-lp4k1VRKPU9eBnPePjnJ9M2RF3i7PC30gXs70+elCVfgwLwx1tv5+ctxdtwxqZa7" crossorigin="anonymous"></script>';
+
+const namedArgs = Yargs
+      .usage('staticrypt <input> <password> options')
+      .demandCommand(2)
+      .default({'output': null, 'title': null, 'instructions': null, 'embed': null})
+      .alias('o', 'output')
+      .alias('t', 'title')
+      .alias('i', 'instructions')
+      .alias('e', 'embed')
+      .argv;
+
+if(namedArgs._.length != 2){
+    Yargs.showHelp();
+    process.exit(1);
+}
+
+const input = namedArgs._[0];
+const password = namedArgs._[1];
```

Comment:
If my password is `123` it is cast to an int and crypto-js will throw an error, so we need to explicitly use a string:

    const password = namedArgs._[1].toString();

Reply:
(for good measure we could do the same for the input)